src/src/rx-serial/SerialSBUS.cpp

```diff
@@ -8,19 +8,23 @@
 #define SBUS_FLAG_SIGNAL_LOSS       (1 << 2)
 #define SBUS_FLAG_FAILSAFE_ACTIVE   (1 << 3)
 
-extern RxConfig config;
-
+const auto UNCONNECTED_CALLBACK_INTERVAL_MS = 10;
 const auto SBUS_CALLBACK_INTERVAL_MS = 9;
 
-uint32_t SerialSBUS::sendRCFrame(bool frameAvailable, uint32_t *channelData)
+uint32_t SerialSBUS::sendRCFrame(bool frameAvailable, bool frameMissed, uint32_t *channelData)
 {
     static auto sendPackets = false;
     if ((failsafe && config.GetFailsafeMode() == FAILSAFE_NO_PULSES) || (!sendPackets && connectionState != connected))
     {
-        return SBUS_CALLBACK_INTERVAL_MS;
+        return UNCONNECTED_CALLBACK_INTERVAL_MS;
```

Comment:
Why has this changed?

Reply:
Just using a more appropriately named constant.